### PR TITLE
Fix sample for amp-form

### PR DESCRIPTION
**Pull request by sebastianbenz**
Post requests are only allowed via xhr.

### CODE REVIEW

File: extensions/amp-form/amp-form.md [hunk 1]
```diff
@@ -48,7 +48,10 @@ The `amp-form` extension **MUST** be loaded if you're using `<form>` or any inpu
 
 Example:
 ```html
-<form method="post" action="https://example.com/subscribe" target="_blank">
+<form method="post" 
+      action="https://example.com/subscribe"
```
Comment (mkhatib) [inline extensions/amp-form/amp-form.md]: Please drop action from here - we don't allow `action` on `post` anymore.

Comment (sebastianbenz) [inline extensions/amp-form/amp-form.md]: Done.

**Comment by sebastianbenz** [comment]
Didn't realise action only works with GET. Updated the docs to reflect this as well.

File: extensions/amp-form/amp-form.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ The `amp-form` extension **MUST** be loaded if you're using `<form>` or any inpu
 
 Example:
 ```html
-<form method="post" action="https://example.com/subscribe" target="_blank">
```
Comment (mkhatib) [inline extensions/amp-form/amp-form.md]: Unfortunately we still require `target` for all. Feel free to file a bug to remove that requirement from `POST` requests but please keep it in the examples for now.

File: extensions/amp-form/amp-form.md [hunk 3]
```diff
@@ -60,28 +60,34 @@ Example:
         </label>
         <input type="submit" value="Subscribe">
     </fieldset>
+    <div submit-success>
+        Subscription successful!
```
Comment (mkhatib) [inline extensions/amp-form/amp-form.md]: This would probably error, you'd need to encapsulate the text with a `template` element.

**Comment by mkhatib** [comment]
Approved with couple more changes.

File: extensions/amp-form/amp-form.md [hunk 4]
```diff
@@ -60,28 +60,34 @@ Example:
...
+        Subscription failed!
+    </div>
 </form>
 ```
 
 ## Attributes
 
 **target**
-__required__
+__required__ for `GET` requests.
```
Comment (dandv) [inline extensions/amp-form/amp-form.md]: [requiring `target` doesn't make sense for XHR requests](https://github.com/ampproject/amphtml/issues/5607#issuecomment-253894602) IMO.

**Comment by mkhatib** [comment]
@sebastianbenz did you manage to update this PR yet?

**Comment by mkhatib** [comment]
@sebastianbenz note that I sent a PR that touches on same path here: #5671

**Comment by mkhatib** [comment]
Closing this for #5986 - thanks @sebastianbenz

**Pull request closed without merging**